airflow/docker-compose.yaml

```diff
@@ -101,6 +101,7 @@ x-airflow-common:
     POD_LOCATION: "us-west2-a"
     AIRFLOW_ENV: "development"
     CALITP_USER: "pipeline"
+    SENTRY_DSN: "http://b5b82f68a88f4bbdb627fa774636cf62@sentry-nginx.sentry.svc.cluster.local/2"
```

Comment:
does this value work in local docker-compose envs?

Reply:
It's passed through to PodOperators that run in the cluster; the _Airflow_ running in docker-compose won't have access until we set up Ingress.

I'll duplicate the env var and add a comment.

Reply:
This now uses the same DSN.